src/System.Runtime.NumericsTestData/System.Runtime.NumericsTestData.csproj

```diff
@@ -0,0 +1 @@
+<Project Sdk="Microsoft.Build.NoTargets" />
```

Comment:
The project's name should be `System.runtime.Numerics.TestData.csproj` so that some of the msbuild conditions kick in correctly.